client/me/help/main.jsx

```diff
@@ -211,14 +221,62 @@ class Help extends React.PureComponent {
 				onClick={ this.trackCoursesButtonClick }
 				href="https://wordpress.com/webinars"
 				target="_blank"
-				title={ this.props.translate( 'Courses' ) }
-				description={ this.props.translate(
+				title={ translate( 'Courses' ) }
+				description={ translate(
 					'Learn how to make the most of your site with these courses and webinars'
 				) }
 			/>
 		);
 	};
 
+	supportSessionCard = () => {
+		const { translate, hasAppointment, scheduleId } = this.props;
+
+		//If we already have an appointment or the scheduleId has not been loaded, bail
+		if ( hasAppointment || null === scheduleId ) {
+			return;
+		}
+
+		return (
+			<Card className="help__support-session-card">
+				<div className="help__support-session-text">
+					<h2 className="help__support-session-title">
+						{ translate( 'Schedule a support session' ) }
+					</h2>
+					<p className="help__support-session-description">
+						{ translate(
+							'Quick Start Support Sessions give you a way to talk to one of our Happiness Engineers via a screen share with audio.'
+						) }
+					</p>
+					<div className="help__support-session-actions">
+						<Button
+							className="help__support-session-action"
+							primary
+							href={ localizeUrl( 'https://wordpress.com/checkout/offer-quickstart-session/' ) }
```

Comment:
Since this is a Calypso URL, I don't think we need to localize it. 
```suggestion
							href="/checkout/offer-quickstart-session"
```